assets/src/components/parts/janus-popup/Popup.tsx

```diff
@@ -51,8 +51,12 @@ const Popup: React.FC<PartComponentProps<PopupModel>> = (props) => {
     // result of init has a state snapshot with latest (init state applied)
     setInitSnapshot(initResult);
     const currentStateSnapshot = initResult.snapshot;
+    const isOpenByDefault: boolean | undefined = currentStateSnapshot[`stage.${id}.openByDefault`];
+    if (isOpenByDefault !== undefined) {
+      setShowPopup(isOpenByDefault);
+    }
     const isOpen: boolean | undefined = currentStateSnapshot[`stage.${id}.isOpen`];
-    if (isOpen !== undefined) {
+    if (isOpen !== undefined && !isOpenByDefault) {
```

Comment:
when the component loads for the first time, if "OpenByDefault" = true then we have to ignore the "isOpen". This only happens for the first time and after that user can change it from trap states.

I have tested this with both the lessons